aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -9,11 +9,6 @@ namespace Assets.Scripts.WFC
     /// </summary>
     public class GridPlacer : MonoBehaviour
     {
-        /// <summary>
-        /// Represents the grid.
-        /// </summary>
-        private Grid _grid;
-
         /// <summary>
         /// The size of the tiles in the x-direction.
         /// </summary>
```

Comment:
Could you please add the s_ prefix to the constants below? ;)

Reply:
I think our wiki actually says it should be PascalCase, but the editorconfig says it should be _camelCase